servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -438,3 +453,66 @@ def test_query_jobs_submission_pagination(self):
                     page_token=response.next_page_token,
                     extensions=ExtendedQueryFields(submission=min_time)),
                 [job2])
+
+        def test_aggregtion_jobs(self):
+            labels = {
+                'aggregation-testing-unique-1': 'testing-rocks',
+                'aggregation-testing-unique-2': 'debugging-bad',
+                'aggregation-testing-unique-3': 'yeah'
+            }
+
+            # Add timestamp to distinguish from old test jobs
+            name = 'aggregation-testing' + str(datetime.datetime.now().minute)
+
+            # Create an aborted job
+            job_aborted = self.api_job_id(
+                self.start_job('sleep 30', labels=labels, name=name))
+            time.sleep(10)
```

Comment:
Generally we'd want to split out provider-specific code into the two different test files (local and google), though it would be unfortunate to duplicate this whole test in both places just to remove the sleep for google.

I'd suggest you add an abstract method to `BaseTestCases` called `start_running_job` and then have a separate local and google implementation. 

Local:
```
def start_running_job(self, command, name=None, labels={}):
  self.start_job(command, name=name, labels=labels):
  time.sleep(10)
  self.wait_status(job_aborted, ApiStatus.RUNNING)
```

Google:
```
def start_running_job(self, command, name=None, labels={}):
  self.start_job(command, name=name, labels=labels):
  self.wait_status(job_aborted, ApiStatus.RUNNING)
```


Then this method can just call `self.start_running_job(...)`. Additionally, the separate `test_abort_job` tests in `test_jobs_controller_local` and `test_jobs_controller_google` can be refactored to use this method and put in this file.